pages/p/ticket/[id].tsx

```diff
@@ -46,6 +45,7 @@ const TicketPage = ({
           content="Mi Ticket para la JSConf Chile. Obtén tu ticket acá!"
         />
         <meta
+          name="image"
```

Comment:
Se probó agregando width y height, Esto según recomendación mía, ya que hace poco solucioné el mismo problema de las imágenes en LinkedIn agregando esas metas

FYI: Puedes leer mas sobre este meta [acá](https://www.semantica.co.za/everything-you-need-to-know-about-open-graph-images-and-why-they-matter/#:~:text=What%20size%20should%20my%20Open,is%20200%20by%20200%20pixels.)